shared/productTypes.ts

```diff
@@ -259,7 +258,6 @@ export type ProductTypeKeys =
  */
 export type GroupedProductTypeKeys =
 	| 'membership'
-	| 'contributions'
```

Comment:
this is where [mdapi defines mmaCategory](https://github.com/guardian/members-data-api/blob/b59dec8c7065a42dd3995907c968d8a382c0efbf/membership-attribute-service/app/models/AccountDetails.scala#L223) and contributions is gone